frontend_lib/test/helper.spec.js

```diff
@@ -375,8 +374,12 @@ describe('helper.js', () => {
     })
 
     describe('if the source is NULL', () => {
-      it('should throw an EXCEPTION_MENTION_PARSING exception', () => {
-        expect(() => wrapMentionsInSpanTags(null)).to.throw(EXCEPTION_MENTION_PARSING)
+      it('should throw an error exception', () => {
+        try {
+          wrapMentionsInSpanTags(null)
+        } catch (e) {
+          expect(typeof e).to.equal('Error')
+        }
```

Comment:
I'm not sure checking that `e` is actually an error.
But there is something fishy here: `typeof new Error()` is `'object'`. If you really want to test that e is an instance of error, (e instanceof Error).should.equal(true)